packages/google_sign_in/google_sign_in_web/example/integration_test/gapi_utils_test.dart

```diff
@@ -7,21 +7,16 @@ import 'package:integration_test/integration_test.dart';
 
 import 'package:google_sign_in_web/src/generated/gapiauth2.dart' as gapi;
 import 'package:google_sign_in_web/src/utils.dart';
-import 'package:mockito/mockito.dart';
```

Comment:
`import 'package:test/fake.dart`?

Reply:
I didn't know where this was coming from, it seemed to come from some implicit import!

Reply:
This was concerning me, so I dug a little bit. Fake now (also) comes from flutter_test as well:

https://master-api.flutter.dev/flutter/flutter_test/flutter_test-library.html

No need to import package:test anymore!

(It's just a little bit "too new", hence the link to the "master" docs :P)